go/embedded/online_features.go

```diff
@@ -0,0 +1,113 @@
+package embedded
+
+import (
+	"context"
+	"github.com/apache/arrow/go/arrow"
+	"github.com/apache/arrow/go/arrow/array"
+	"github.com/apache/arrow/go/arrow/cdata"
+	"github.com/feast-dev/feast/go/internal/feast"
+	"github.com/feast-dev/feast/go/protos/feast/types"
+	"github.com/feast-dev/feast/go/utils"
+	"log"
+)
+
+type OnlineFeatureService struct {
+	fs *feast.FeatureStore
+}
+
+type OnlineFeatureServiceConfig struct {
+	RepoPath   string
+	RepoConfig string
+}
+
+type DataTable struct {
+	DataPtr   uintptr
+	SchemaPtr uintptr
+}
+
+func NewOnlineFeatureService(conf *OnlineFeatureServiceConfig) *OnlineFeatureService {
+	repoConfig, err := feast.NewRepoConfigFromJSON(conf.RepoPath, conf.RepoConfig)
+	if err != nil {
+		log.Fatalln(err)
+	}
+
+	fs, err := feast.NewFeatureStore(repoConfig)
+	if err != nil {
+		log.Fatalln(err)
+	}
+	defer fs.DestructOnlineStore()
+	return &OnlineFeatureService{fs: fs}
+}
+
+func (s *OnlineFeatureService) GetOnlineFeatures(
+	featureRefs []string,
+	featureServiceName string,
+	entities DataTable,
+	fullFeatureNames bool,
+	projectName string,
+	output DataTable) error {
+
+	entitiesRecord, err := readArrowRecord(entities)
```

Comment:
Would you mind dropping some comments in this function? I think it would be pretty informative for contributors to understand how it functions and how it will be used.

Reply:
Added some comments (see fun definition)